scripts/zones/Aydeewa_Subterrane/mobs/Pandemonium_Warden.lua

```diff
@@ -32,9 +87,12 @@ end;
 
 function onMobEngaged(mob,target)
     -- pop pets
-    for i = 17056170, 17056177, 1 do
-        SpawnMob(i,180):updateEnmity(target);
-        GetMobByID(i):setModelId(1841);
+    for i = 1, 8 do
+        local pet = GetMobByID(petIDs[1][i]);
+        pet:setModelId(1841);
+        pet:spawn();
+        pet:updateClaim(target);
```

Comment:
I was talking about this else block (kinda hard to see cause all the old code is mixed up in it): https://github.com/DarkstarProject/darkstar/pull/3303/files#diff-024cb4c707543574f17510da7bec95dcR155

I'll remove that claim stuff, was just trying to prevent any add shenanigans because the current version has buggy adds which caused a lot of drama on our server. I need to test it as a non GM too, I think GM status keeps them from aggroing.